docs/user-guide/src/troubleshooting.md

```diff
@@ -88,3 +88,151 @@ of a forced deletion of its BareMetalHost object. If valid BMC credentials were
 provided, Ironic will keep checking the power state of the host and enforcing
 the last requested power state. The only solution is again to delete the
 Ironic's internal database.
+
+## BMH registration errors
+
+BMC credentials could be wrong or missing. These issues show up in the
+BareMetalHost’s status and as Events.
+
+Check both `kubectl describe bmh <name>` and recent Events for details.
+
+Example output:
+
+```text
+Normal  RegistrationError  23s   metal3-baremetal-controller  Failed to get 
+power state for node 67ac51af-a6b3. Error: Redfish exception occurred. 
+Error: HTTP GET https://192.168.111.1:8000/redfish/v1/Systems/... returned code 401.
+```
+
+## BMH inspection errors
+
+### The host is not able to communicate back results to Ironic
+
+If the host cannot communicate with Ironic, it will result in a timeout. Access
+to serial logs is needed to determine the exact issue.
+
+Example output from `kubectl get bmh -A`:
+
+```text
+NAMESPACE   NAME     STATE        CONSUMER   ONLINE   ERROR              AGE
+metal3      node-1   inspecting             true     inspection error   46m
+```
+
+BareMetalHost's events from `kubectl describe bmh <name> -n <namespace>`:
+
+```text
+Events:
+  Type    Reason              Age    From                         Message
+  ----    ------              ----   ----                         -------
+  Normal  InspectionStarted   37m    metal3-baremetal-controller  Hardware inspection started
+  Normal  InspectionError     7m12s  metal3-baremetal-controller  timeout reached while inspecting the node
+```
+
+### Incompatible configuration
+
+This can happen when trying to use virtual media or UEFI when not supported.
+The error will show in status and events.
```

Comment:
```suggestion
This can occur when attempting to use virtual media or UEFI on hardware
that does not support it. The error will show in status and in events.
```